src/metabase/util/date_2.clj

```diff
@@ -236,7 +240,7 @@
   ([unit]
    (extract (t/zoned-date-time) unit))
 
-  ([t :- Temporal, unit :- (apply s/enum extract-units)]
+  ([t :- Temporal unit :- (apply s/enum extract-units)]
```

Comment:
unrelated change?

Reply:
Yeah, but I've been trying to clean up stuff like this when I come across it. A comma doesn't belong here